go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/xeipuuv/gojsonschema
 
+go 1.14
```

Comment:
doesn't seem to match your commit message. why do you need to change it?

Reply:
You must have commented as I was updating. This is now out of date